rooms.js

```diff
@@ -261,6 +261,22 @@ class Room {
 			}
 		}
 	}
+
+	get metadata() {
+		let settings = {
+			// settings
+			privacy: (this.isPrivate === true ? 'secret' : this.isPrivate === 'hidden' ? 'hidden' : 'public'),
+			modchat: this.modchat,
+			modjoin: this.modjoin,
+
+			// games
```

Comment:
why not just send room.game.title, if room.game is a thing

that way, you don't have to send it for each specific game